nbs/core.ipynb

```diff
@@ -3204,6 +3321,147 @@
     "        nf.fit(AirPassengersPanel_train)\n",
     "        assert any(\"ignoring lr_scheduler_kwargs as the lr_scheduler is not specified\" in str(w.message) for w in issued_warnings)\n"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0441e9d5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#| hide\n",
+    "# test conformal prediction, method=conformal_distribution\n",
+    "\n",
+    "conformal_intervals = ConformalIntervals()\n",
+    "\n",
+    "models = []\n",
+    "for nf_model in [NHITS, RNN, StemGNN]:\n",
```

Comment:
Please replace `StemGNN` with `TSMixer` everywhere where you are using `StemGNN` (it's another Multivariate model, but it's way faster than `StemGNN`, so this speeds up the tests)